packages/integrations/markdoc/README.md

```diff
@@ -401,6 +401,36 @@ const { Content } = await entry.render();
 
 This can now be accessed as `$frontmatter` in your Markdoc.
 
+## Integration config options
+
+The Astro Markdoc integration handles configuring Markdoc options and capabilities that are not available through the `markdoc.config.js` file.
+
+### allowHTML
+
+By default, Markdoc will **NOT** recognize HTML markup as semantic content (this differs from Markdown, which does support HTML markup by default).
+
+You can enable an Astro provided feature via the `allowHTML` integration option which will enable HTML parsing in Markdoc markup.
+
+Enabling this feature will allow you to interleave Markdoc tags and nodes with HTML markup seamlessly.
+
+> **Warning**
+> When `allowHTML` is enabled, all Markdoc markup will be processed through the [htmlparser2](https://www.npmjs.com/package/htmlparser2) library.
+> 
+> HTML markup inside Markdoc documents will be rendered as actual HTML elements (this includes `<script>`), making attack vectors like XSS possible.
+> 
+> Care should be taken that rendered Markdoc + HTML markup comes from trusted sources. 
```

Comment:
```suggestion
> **Warning**
> When `allowHTML` is enabled, HTML markup inside Markdoc documents will be rendered as actual HTML elements (this includes `<script>`), making attack vectors like XSS possible.
>
> Ensure that any HTML markup comes from trusted sources. 
```

Excellent detail here! I just removed the htmlparser2 notice to avoid burying the lead, and to remove technical details that aren't needed to understand the problem. I'll differ to @sarah11918 on whether that's worth bringing back.

Reply:
Agree with @bholmesdev here! I approve his suggestion!

nit: change `this includes` to `including` for all the brownie points!  :cookie: 

(Will also do a tiny review of the earlier text, so expect that incoming!)